app/views/field_slips/_form.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= form_with(model: field_slip) do |form| %>
+  <% if field_slip.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(field_slip.errors.count, "error") %> prohibited this field_slip from being saved:</h2>
+
+      <ul>
+        <% field_slip.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
```

Comment:
I think in this situation we've been using `<div class="alert alert-danger">` (instead of `<div style="color: red">`)